scripts/lib/jsonManager.js

```diff
@@ -19,7 +20,7 @@ export function saveWeekLogs(year, week, issues) {
   const yearLog = logs(year);
   const weekIssues = yearLog.get(weekPath);
   if (weekIssues) {
-    yearLog.set(weekPath, uniqBy([...weekIssues, ...issues], "id"));
+    yearLog.set(weekPath, unionBy([...issues, ...weekIssues], "id"));
```

Comment:
If you pass only one array, `uniqBy` and `unionBy` seems to have the same behavior. The only difference is that you can pass multiple arrays to `union`, ie `unionBy(issues, weekIssues, "id")`.

As per https://lodash.com/docs/4.17.15:
> Union - Creates an array of unique values, in order, **from all given arrays** using SameValueZero for equality comparisons.
> Uniq - Creates a duplicate-free version **of an array**, using SameValueZero for equality comparisons, in which only the first occurrence of each element is kept. The order of result values is determined by the order they occur in the array.

Unless I'm missing something, I'd say we should stick with `uniqBy`.

Reply:
@Betree : You are right; I was confused here. I was testing with unionBy(issues, weekIssues, "id") and thought I checked that in somehow. :thinking: